backend/static/backend/scripts/event_detail.js

```diff
@@ -74,8 +75,8 @@ postButton.addEventListener("click", function () {
   const rating = document.querySelector('input[name="rating"]:checked').value;
 
   const formData = new FormData();
-  formData.append("rating", rating);
-  formData.append("review_text", reviewText);
+  formData.append('rating', rating);
+  formData.append('review_text', reviewText);
```

Comment:
[nits] Not sure why this is not enforced by black, but if possible can we keep it as double quotes?